packages/wormhole-sdk/package.json

```diff
@@ -38,6 +39,7 @@
     "eslint-plugin-simple-import-sort": "^7.0.0",
     "eslint-plugin-sonarjs": "^0.12.0",
     "eslint-plugin-unused-imports": "^2.0.0",
+    "ethereum-waffle": "^3.4.0",
```

Comment:
sidenote: this dependency is the worst -- it has dependency on ton of unrelated things including old version of typechain :|

Reply:
Good to know. We can definitely get rid of it, it's only used to make expect() work with ether's BigNumber which is not critical

Reply:
Maybe we can use this intead? https://www.npmjs.com/package/@ethereum-waffle/chai